### ISSUE TITLE
Summary for search results shows wrong percentages

### ISSUE THREAD

**Issue by Schnark** [Fyrd/caniuse#5190]
E.g. https://caniuse.com/#search=split

This shows 9 results. All have full support in Firefox 70, but the summary claims it has only 50 % support for all these features. According to the summary, support last increased between 60 and 63, but the data don't have such a change.
Chrome 78 has at least partial support for all (full support for 7, one prefixed, one partial), with no difference in version 79, but the summary says 42 %, while version 79 will raise to 50 %.
Current and future Safari has similar support as Chrome (except missing data for ChannelSplitterNode() constructor), but has only 8 % in the summary, which will increase to 25 %.
Data for UC Browser and KaiOS Browser are identical, but the summary has 33 % for UC Browser, and only 17 % for KaiOS.

If I ignore the results from MDN, and toss a coin for String.prototype.split (as it doesn't have a table), the summary matches the data a bit better, but still doesn't make much sense. So, something is broken with the way the percentages in the summary are calculated.

**Comment by Schnark** [comment]
Other example: https://caniuse.com/#search=lazy
There are 2 features, Firefox supports neither, yet in summary has 40 % support. Edge will raise from 0 % to 40 % by dropping support.

**Comment by Fyrd** [comment]
Yeah, I feel the summary's currently too broken and confusing to be useful, so have disabled it for now. I'm in the middle of rewriting the front-end code so will make sure it works as expected once that's complete.
